app/models/have_a_degree.rb

```diff
@@ -1,31 +1,46 @@
 class HaveADegree < Base
+  attribute :degree_options, :string
   attribute :degree_status_id, :string
   attribute :degree_type_id, :string
 
+  before_validation :set_degree_status
   before_validation :set_degree_type
 
-  OPTIONS = { yes: "222750000", no: "222750004", studying: "studying", equivalent: "222750005" }.freeze
+  DEGREE_OPTIONS = { yes: "yes", no: "no", studying: "studying", equivalent: "equivalent" }.freeze
+  DEGREE_STATUS_OPTIONS = { yes: "222750000", no: "222750004", studying: "studying" }.freeze
```

Comment:
I think this is going to be confusing when we come back to it later - would it be cleaner if this step only had the `DEGREE_OPTIONS` and the `degree_status_id` and `degree_type_id` were set as part of the submission/where ever you build the payload based on `degree_options`?

Reply:
It is quite complex, but we have to put the logic somewhere, I think the attributes should be set in the model, rather than merged later, e.g in the candidate_submission service

Reply:
That's fair; can we have some indication that the values that aren't integer ids (so `studying`/`equivalent`) are going to be set in subsequent steps though? even if it's just a comment

Reply:
We should put it in the READme, or wiki.